### ISSUE TITLE
Outdated assembly reference in Interactive.fsx for Nu.Gaia

### ISSUE THREAD

**Issue by vtquan** [bryanedds/Nu#284]
Possibly due to #274. In Interactive.fsx for Nu.Gaia, assembly reference "../../Nu/Nu.Gaia.Design/bin/Debug/Nu.Gaia.Design.exe" would not be found. It would be in "../../Nu/Nu.Gaia.Design/bin/x64/Debug/Nu.Gaia.Design.exe" instead.

**Comment by bryanedds** [comment]
Thank you, I'll patch this up!

**Comment by bryanedds** [comment]
Okay, there were a few issues here that I had to fix. Primarily, Gaia wasn't intended to run from the REPL and this file was just for testing. However, I could see how this would cause confusion, so I made it run Gaia as usual.

I haven't tested this way of running Gaia too much, however, so I can't guarantee there won't be basic issues. The recommended way to run Gaia is as a normal program either from Visual Studio / VSCore, or from the command line or explorer.
